src/hooks/useWatchPendingTxs.ts

```diff
@@ -47,7 +46,7 @@ export const useWatchPendingTransactions = ({ address }: { address: string }) =>
           testnetMode: !!connectedToHardhat,
         })
       );
-      queryClient.invalidateQueries(nftsQueryKey({ address, sortBy: getNftSortForAddress(address) }));
+      invalidateAddressNftsQueries(address);
```

Comment:
does this work?

Reply:
I may be missing context but doesn't it makes sense invalidate all address nft queries in this `refreshAssets`? 
when invalidating only the currently selected sort type it the others will still be cached so switching the sort will probably return old data

Reply:
yeah ideally we'd want to refresh all, but I don't think this will achieve that. The query key has to match the cached data in order to invalidate it. So we'd want to loop over all options and invalidate those individually.